prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/LA/LA-receiver-transform.yml

```diff
@@ -31,13 +31,38 @@ elements:
     value: [ '"LA-ELR"' ]
     hl7Spec: [ 'MSH-5-1' ]
 
+  - name: la-receiving-application-universal-id
+    condition: 'true'
+    value: [ '"LA-ELR"' ]
+    hl7Spec: [ 'MSH-5-2' ]
+
+  - name: la-receiving-application-universal-id-type
+    condition: 'true'
+    value: [ '"ISO"' ]
+    hl7Spec: [ 'MSH-5-3' ]
+
   - name: la-receiving-facility-namespace-ID
     condition: 'true'
     value: [ '"LADOH"' ]
     hl7Spec: [ 'MSH-6-1' ]
 
+  - name: la-receiving-facility-universal-id
+    condition: 'true'
+    value: [ '"LADOH"' ]
+    hl7Spec: [ 'MSH-6-2' ]
+
+  - name: la-receiving-facility-universal-id-type
+    condition: 'true'
+    value: [ '"ISO"' ]
+    hl7Spec: [ 'MSH-6-3' ]
+
   - name: la-patient-race-coding-system
     resource: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value.coding'
     condition: '%resource.code.exists()'
     value: [ '%resource.system.getCodingSystemMapping()' ]
-    hl7Spec: [ '%{PID}-10-3' ]
\ No newline at end of file
+    hl7Spec: [ '%{PID}-10-3' ]
+
+  - name: la-patient-county-codes
```

Comment:
This casues an exception if the lookup value can't be found:

"message":"gov.cdc.prime.router.fhirengine.translation.hl7.schema.ConfigSchemaElementProcessingException: Error encountered while applying: la-patient-county-codes in /metadata/hl7_mapping/receivers/STLTs/LA/LA-receiver-transform.yml to FHIR bundle. \nError was: convertToString(...) must not be null","thread":"pool-2-thread-38","timestamp":"2024-08-12T16:38:11.477Z","level":"ERROR","logger":"gov.cdc.prime.router.fhirengine.engine.FHIRTranslator"}
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z] Executed 'Functions.translate-fhir' (Failed, Id=8dccbdf7-fa01-4d99-a372-20e91d92fb3e, Duration=633ms)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z] System.Private.CoreLib: Exception while executing function: Functions.translate-fhir. System.Private.CoreLib: Result: Failure
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z] Exception: NullPointerException: convertToString(...) must not be null
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z] Stack: java.lang.reflect.InvocationTargetException
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at java.base/java.lang.reflect.Method.invoke(Method.java:568)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at com.microsoft.azure.functions.worker.broker.JavaMethodInvokeInfo.invoke(JavaMethodInvokeInfo.java:22)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at com.microsoft.azure.functions.worker.broker.JavaMethodExecutor.execute(JavaMethodExecutor.java:22)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at com.microsoft.azure.functions.worker.chain.FunctionExecutionMiddleware.invoke(FunctionExecutionMiddleware.java:19)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at com.microsoft.azure.functions.worker.chain.InvocationChain.doNext(InvocationChain.java:21)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at com.microsoft.azure.functions.worker.broker.JavaFunctionBroker.invoke(JavaFunctionBroker.java:133)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at com.microsoft.azure.functions.worker.broker.JavaFunctionBroker.invokeMethod(JavaFunctionBroker.java:124)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at com.microsoft.azure.functions.worker.handler.InvocationRequestHandler.execute(InvocationRequestHandler.java:34)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at com.microsoft.azure.functions.worker.handler.InvocationRequestHandler.execute(InvocationRequestHandler.java:10)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at com.microsoft.azure.functions.worker.handler.MessageHandler.handle(MessageHandler.java:44)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at com.microsoft.azure.functions.worker.JavaWorkerClient$StreamingMessagePeer.lambda$onNext$0(JavaWorkerClient.java:94)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at java.base/java.lang.Thread.run(Thread.java:840)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z] Caused by: gov.cdc.prime.router.fhirengine.translation.hl7.schema.ConfigSchemaElementProcessingException: Error encountered while applying: la-patient-county-codes in /metadata/hl7_mapping/receivers/STLTs/LA/LA-receiver-transform.yml to FHIR bundle. 
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z] Error was: convertToString(...) must not be null
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.translation.hl7.FhirToHl7Converter.processSchema(FhirToHl7Converter.kt:164)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.translation.hl7.FhirToHl7Converter.processSchema$default(FhirToHl7Converter.kt:142)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.translation.hl7.FhirToHl7Converter.process(FhirToHl7Converter.kt:90)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.engine.FHIRTranslator.getHL7MessageFromBundle$prime_router(FHIRTranslator.kt:227)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.engine.FHIRTranslator.getByteArrayFromBundle$prime_router(FHIRTranslator.kt:200)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.engine.FHIRTranslator.sendTranslated$prime_router(FHIRTranslator.kt:148)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.engine.FHIRTranslator.doWork(FHIRTranslator.kt:84)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.engine.FHIREngine.run(FHIREngine.kt:224)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.azure.FHIRFunctions$runFhirEngine$newMessages$1.invoke(FHIRFunctions.kt:197)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.azure.FHIRFunctions$runFhirEngine$newMessages$1.invoke(FHIRFunctions.kt:196)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.azure.DatabaseAccess.transactReturning$lambda$1(DatabaseAccess.kt:107)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at org.jooq.impl.DefaultDSLContext.lambda$transactionResult0$3(DefaultDSLContext.java:531)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at org.jooq.impl.Tools$3$1.block(Tools.java:6068)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at java.base/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3465)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at java.base/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3436)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at org.jooq.impl.Tools$3.get(Tools.java:6065)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at org.jooq.impl.DefaultDSLContext.transactionResult0(DefaultDSLContext.java:579)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at org.jooq.impl.DefaultDSLContext.transactionResult(DefaultDSLContext.java:503)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.azure.DatabaseAccess.transactReturning(DatabaseAccess.kt:107)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.azure.FHIRFunctions.runFhirEngine(FHIRFunctions.kt:196)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.azure.FHIRFunctions.doTranslate(FHIRFunctions.kt:169)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.azure.FHIRFunctions.doTranslate$default(FHIRFunctions.kt:163)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.azure.FHIRFunctions.translate(FHIRFunctions.kt:154)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  ... 19 more
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z] Caused by: java.lang.NullPointerException: convertToString(...) must not be null
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.translation.hl7.utils.FhirPathUtils.evaluateString(FhirPathUtils.kt:228)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.translation.hl7.FhirToHl7Converter.getValueAsString$prime_router(FhirToHl7Converter.kt:117)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.translation.hl7.FhirToHl7Converter.processElement$prime_router(FhirToHl7Converter.kt:225)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  at gov.cdc.prime.router.fhirengine.translation.hl7.FhirToHl7Converter.processSchema(FhirToHl7Converter.kt:162)
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z]  ... 41 more
2024-08-12 09:38:11 [2024-08-12T16:38:11.479Z] .
2024-08-12 09:38:11 [2024-08-12T16:38:11.928Z] Executing 'Functions.translate-fhir' (Reason='New queue message detected on 'elr-fhir-translate'.', Id=3fab637b-36ff-4a49-b7bc-e3839f110e35)
2024-08-12 09:38:11 [2024-08-12T16:38:11.928Z] Trigger Details: MessageId: 96211148-bac7-4f64-af88-778ef36ab3eb, DequeueCount: 5, InsertedOn: 2024-08-12T16:38:07.000+00:00